### PR TITLE
feat: add vintage story support

### LINKED ISSUE #328: Add Vintage Story support
Any? :) Thanks, i used the bot before in other games and i love it.

### PR DESCRIPTION
**Pull request by CosminPerRam**
Closes #328
Adds the support via a master server.

### CODE REVIEW

File: GAMES_LIST.md
Comment (cetteup) [inline GAMES_LIST.md:1]: **question:** I presume these seemingly unrelated changes were generated by running the game list script?

File: protocols/vintagestory.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import Core from './core.js'
+import vintagestorymaster from './vintagestorymaster.js'
+
+export default class vintagestory extends Core {
+  async run (state) {
+    const master = new vintagestorymaster()
+    master.options = this.options
+    const masterState = await master.runOnceSafe()
+    const servers = masterState.raw.servers
+    console.log(`${this.options.address}:${this.options.port}`)
```
Comment (cetteup) [inline protocols/vintagestory.js:10]: **todo:** Remove debugging code.

**Review by cetteup — CHANGES_REQUESTED** [review]

File: GAMES_LIST.md
Comment (CosminPerRam) [reply GAMES_LIST.md:1]: We keep fumbling the document (i.e. place the game manually wrongfuly or forgot to alter mentions), this could be easily fixed by having CI to check/automatically alter the file.

Comment (CosminPerRam) [reply GAMES_LIST.md:1]: Made #607 in light of this issue.
